server/models.py

```diff
@@ -104,6 +94,14 @@ class ResourcesNeeded(db.Model):
         ForeignKey("resource.id"), nullable=False)
 
 
+class LoggedEvent(db.Model):
```

Comment:
IMO the entirety of the logic could be handled in this class as in the other Event class is not needed.

Reply:
IMO this is a db model. Therefore it shall have no method specification and defines only the table. To mix both contexts would by confusing.

Reply:
I have the same opinion like @YIllmann.